ga4gh/cli.py

```diff
@@ -153,94 +178,159 @@ def __init__(self, args):
         self.setHttpClient(request, args)
 
     def run(self):
-        self._run('searchVariants', 'id')
+        self._run(self._httpClient.searchVariants, 'id')
 
 
-class ReferenceSetSearchRunner(AbstractSearchRunner):
+class SearchReferenceSetsRunner(AbstractSearchRunner):
     """
     Runner class for the referencesets/search method.
     """
+    class GASearchReferenceSetsRequestGoogle(protocol.ProtocolElement):
+
+        __slots__ = ['accessions', 'md5checksums', 'pageSize', 'pageToken']
+
+        def __init__(self):
+            self.accessions = []
+            self.md5checksums = []
+            self.pageSize = None
+            self.pageToken = None
+
     def __init__(self, args):
-        super(ReferenceSetSearchRunner, self).__init__(args)
-        request = protocol.GASearchReferenceSetsRequest()
-        setCommaSeparatedAttribute(request, args, 'accessions')
-        setCommaSeparatedAttribute(request, args, 'md5checksums')
+        super(SearchReferenceSetsRunner, self).__init__(args)
         if self.usingWorkaroundsFor(client.HTTPClient.workaroundGoogle):
```

Comment:
In general, for this type of situation I like the form
```python
request = protocol.GASearchReferenceSetsRequest()
if self.usingWorkaroundsFor(client.HTTPClient.workaroundGoogle):
    request = self.GASearchReferenceSetsRequestGoogle()
```
I think it communicates the default state a bit better, and it saves a line of code. OK, we allocate an object that we don't use, but this is immaterial in the larger scheme of things.

Feel completely free to ignore this, I'm just making a comment!